projectmanagement/src/main/java/life/qbic/projectmanagement/application/sample/SamplePreview.java

```diff
@@ -79,6 +82,9 @@ private SamplePreview(ExperimentId experimentId, SampleId sampleId, String sampl
     this.species = species;
     this.specimen = specimen;
     this.analyte = analyte;
+    // optional columns
+    this.comment = comment == null ? "" : comment;
```

Comment:
```suggestion
    this.comment = comment;
```
Please make the field nullable and change the getter. This is a good example of where the optional type is intended to be used.